sql/core/src/main/scala/org/apache/spark/sql/parquet/newParquet.scala

```diff
@@ -106,12 +106,12 @@ class DefaultSource
       ParquetRelation.createEmpty(
         path,
         data.schema.toAttributes,
-        false,
+        true,
```

Comment:
Use named parameter for allowExisting. 

Also, seems we will overwrite the metadata file. In the case of Append, do we still want to overwrite the metadata?